### PR TITLE
Added new NIDs and new headers

### PR DESCRIPTION
**Pull request by TheOfficialFloW**

### CODE REVIEW

File: include/psp2/npdrm.h [hunk 1]
```diff
@@ -0,0 +1,61 @@
+/**
+ * \usergroup{SceNpdrm}
+ * \usage{psp2/npdrm.h,SceNpdrm_stub}
```
Comment (d3m3vilurr) [inline include/psp2/npdrm.h]: * SceNpdrm -> SceNpDrm
* SceNpdrm_stub -> SceNpDrm_stub

**Comment by devnoname120** [comment]
Please add `SceVideoExport` in `docs/definitions.dox`: https://travis-ci.org/vitasdk/vita-headers/builds/245982681#L197

**Comment by devnoname120** [comment]
Looks good to me.

Travis is failing because it fails to download the sdk but this is fixed by #199. Once it's merged, you will need to rebase on master so that Travis can be run again.

**Comment by TheOfficialFloW** [comment]
Can thisbe merged now?

**Pull request merged**